todo/src/components/TodoList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Todo from './Todo';
+import { connect } from 'react-redux';
+import { toggleTodo, deleteTodo } from "../actions";
+
+const TodoList = (props) => {
+
+    return (
+        <div className="todoList">
+            <ul>
+                {props.todos.map((item, index) => {
+                    return <Todo styleTodo={{ textDecoration: item.completed ? 'line-through' : 'none' }} click={() => props.toggleTodo(item.id)} deleteClick={() => props.deleteTodo(item.id)} todoItem={item.task} key={item.id} />
+                })}
+            </ul>
+        </div>
+    )
+}
+
+const mapStateToProps = (state) => {
```

Comment:
You actually didn't have to use mapStateToProps in this file since you brought in the todos array in your App.js file. All you need to do from App.js is pass the todos array as props to this component. In the connect function, you would just set the mapStateToProps argument as null: `connect(null, { ...action-creators })(component)`